### ISSUE TITLE
[Bug] ServerResponseException: `array_keys($responseHeader, 'Retry-After')`  always return empty.

### ISSUE THREAD

**Issue by quyle92** [code-lts/Laravel-FCM#19]
Hi,
I noticed that there is a bug at `ServerResponseException.php` on line 28: `array_keys($responseHeader, 'Retry-After')` will always return `[]` even if `Retry-After` header is present. 
**Proposed solution**: It should be `array_key_exists('Retry-After', $responseHeader)` so that it can return `true` when `Retry-After` header is present.

**Comment by williamdes** [comment]
Hi @quyle92 !
Thank you so much for reporting this, I am working on fixing this issue and writing tests.

I conclude that `$this->retryAfter = $responseHeader['Retry-After'][0];` is also necessary because it would return an array with the current code.
Can you confirm that ?
Could you send me some example response headers ?

**Comment by williamdes** [comment]
I will create a release as soon as possible, will probably wait for you to give me some feedback about my patch before releasing

**Comment by quyle92** [comment]
@williamdes yes, I'd agree that`$this->retryAfter = $responseHeader['Retry-After'][0];` is also necessary.
I simulated the response by this code:
```php
$retry_after = 30;
$response = new Response(503, ['Retry-After' => $retry_after]);
$ex = new ServerResponseException($response);
dd($ex);
```

**Comment by williamdes** [comment]
Released as [1.7.1](https://github.com/code-lts/Laravel-FCM/releases/tag/1.7.1) 🎉 🚀
